src/core/songloader.cpp

```diff
@@ -252,7 +254,7 @@ SongLoader::Result SongLoader::LoadLocalAsync(const QString& filename) {
 
     SongList song_list = cue_parser_->Load(&cue, matching_cue,
                                            QDir(filename.section('/', 0, -2)));
-    for (Song song : song_list) {
+    for (Song& song : song_list) {
```

Comment:
Avoid non-const references.

Reply:
It should really be also using `qAsConst()` but that is only available since Qt5.7.
Shall I replace it with
```
#if (QT_VERSION < QT_VERSION_CHECK(5, 7, 0))
for (const Song& song : song_list) 
#else
for (const Song& song : qAsConst(song_list)) 
#endif
```
or simply not bother the CLazy warnings and simply use
```
for (const Song& song : song_list) 
```
since Clementine's code generates quite a few more of those CLazy warnings...

Reply:
`for (const Song& song : song_list) ` is fine.